composer.json

```diff
@@ -1,19 +1,22 @@
 {
   "name": "k10r/project",
-  "type": "shopware-core-plugin",
-  "description": "Automatic project-update management",
+  "type": "shopware-plugin",
+  "description": "Automatic project management",
   "license": "MIT",
   "authors": [
     {
-      "name": "Siegmund Mioduszewski",
-      "email": "s.mioduszewski@k10r.de"
-    },
-    {
-      "name": "Uwe Kleinmann",
-      "email": "u.kleinmann@k10r.de"
+      "name": "Kellerkinder GmbH",
+      "email": "hallo@kellerkinder.de"
```

Comment:
We use `anfragen@` for other plugins and in the plugin builder :)

Reply:
Good catch, changed.